include/odp/api/spec/schedule_types.h

```diff
@@ -79,6 +79,9 @@ extern "C" {
  * requests another event from the scheduler, which implicitly releases the
  * context. User may allow the scheduler to release the context earlier than
  * that by calling odp_schedule_release_atomic().
+ * When scheduler is enabled as flow-aware, the event flow hash value affects
+ * scheduling of the event and synchronization is maintained per flow within
+ * each queue.
```

Comment:
hash -> id

"... the event flow hash value affects ..." -> "... the event flow id value affects ..."